openvdb_points/tools/AttributeArray.h

```diff
@@ -150,7 +150,6 @@ struct UnitVecAttributeCodec
 
 ////////////////////////////////////////
 
-
```

Comment:
Can we remove these formatting changes from these commits? Two new lines between classes is consistent.